package.json

```diff
@@ -7,7 +7,9 @@
     "build": "npm run clean && tsc -p tsconfig.json && node src/fileCopy.js",
     "clean": "rimraf dist",
     "deploy:local": "npm run build && cd dist && npm pack",
-    "test": "svc-tools test unit"
+    "test": "svc-tools test unit",
+    "fix": "svc-tools fix"
```

Comment:
add `check` and the pre-commit stuff? Latter is pretty simple, can copy from layouts/FDC3.

Reply:
Done